articles/linear-regression-code/index.md

```diff
@@ -0,0 +1,135 @@
+In the previous article, we looked at the theory behind linear regression. In this article, we will implement the linear regression algorithm from scratch and understand the various steps involved. In my earlier article, I used the in-built function in scikit-learn to predict the houses' prices. Towards the end of the article, we will compare the two approaches and reason the outputs.
+
+### Introduction
+
+As discussed in the previous article, linear regression is a type of supervised learning algorithm. It is used in many applications in the financial industry. First, let us understand the various functions needed to implement a linear regression class, to begin with the coding aspect. 
+
```

Comment:
Suggestion: You could have an additional sub-heading where you define terms such as Bias, Weights, Loss, Independent Variable, Dependent Variable, etc. (Just a single line definition)

I am aware that this article is supposed to be a continuation of another article, but if a beginner to ML is reading this, a glossary would be really helpful for reference.